packages/styles/tag-button.css

```diff
@@ -0,0 +1,12 @@
+.TagButton {
+  vertical-align: middle;
+}
+
+.TagButton__icon {
+  margin-left: var(--space-half);
+}
+
+.TagButton__icon svg {
+  width: var(--tag-button-icon-size);
+  height: var(--tag-button-icon-size);
```

Comment:
This variable should be local to this file and not global (as in variables.css).

Additionally, this doesn't appear to be the right size. The icon looks super tiny in the preview:
![](https://github.com/dequelabs/cauldron/assets/1062039/f2600d0f-a230-41cc-b9be-6d757ae41ac8)